api/src/main/java/io/strimzi/api/kafka/model/KafkaConnectResources.java

```diff
@@ -47,6 +47,17 @@ public static String metricsAndLogConfigMapName(String clusterName) {
         return clusterName + "-connect-config";
     }
 
+    /**
+     * Returns qualified name of the service which works across different namespaces.
+     *
+     * @param clusterName   Name of the Connect CR
+     * @param namespace     Namespace of the Connect deployment
+     * @return              qualified namespace in the format "&lt;service-name&gt;.&lt;namespace&gt;.svc"
+     */
+    public static String qualifiedServiceName(String clusterName, String namespace) {
```

Comment:
We should at least add something to the javadoc of `serviceName()` to say that's it's the namespace-local name.

And we should consider deprecating it and adding a new `localServiceName()` method instead to make the distinction more obvious.

Reply:
OK, fair points.